examples/io/io_usage.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+
+from dffml.noasync import train
+from dffml import Features, DefFeature
+from dffml.operation.output import GetSingle
+from dffml.df.memory import MemoryOrchestrator
+from dffml.df.types import DataFlow, Input, InputFlow
+from dffml.operation.model import model_predict, ModelPredictConfig
+from dffml.operation.io import AcceptUserInput, print_output
+
+from dffml_model_scikit import LinearRegressionModel
+
+scikit_model = LinearRegressionModel(
+    features=Features(
+        DefFeature("Years", int, 1),
+        DefFeature("Expertise", int, 1),
+        DefFeature("Trust", float, 1),
+    ),
+    predict=DefFeature("Salary", int, 1),
+)
+
+# TODO Training should be in different file, only prediction
+# should be here?
+# TODO decide on the stdin input format, if its like {"Years": 0, "Expertise": 1, "Trust": 0.1, "Salary": 10}
+# then literal_eval or something.. ask John
```

Comment:
- Make another operation, `dffml/operation/preprocess.py` for the `literal_eval` operation.
- For the flow, connect `modelprediction.features` to `dffml.mapping.create`
  - Connect `dffml.mapping.create`  value to `literal_eval`
  - Add to dataflow.seed, an `Input` with the `origin="seed.feature"`
```python
        "create_feature_map": InputFlow(
            inputs={"key": ["seed.feature"], ... literal_eval connection for value }
        ),
```

- Change the demo to use the slr model in `dffml/model/slr.py`
  - That way it only needs one piece of input feature data.